idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -140,12 +138,18 @@ class PackedColumnData(UnitModelBlockData):
         default=250,
```

Comment:
Should these be config arguments, or are they just default values for design variables the user should be fixing later?